python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -452,6 +452,15 @@ def test_delete_version(arctic_library):
     assert lib["symbol"].metadata == {"very": "interesting"}
 
 
+def test_non_existent_list_versions_latest_only(arctic_library):
+    lib = arctic_library
+    assert len(lib.list_versions("symbol", latest_only=True)) == 0
+    df = pd.DataFrame({"col1": [1, 2, 3], "col2": [4, 5, 6]})
+    lib.write("symbol2", df)
+    lib.delete("symbol2")
+    assert len(lib.list_versions("symbol2", latest_only=True)) == 0
```

Comment:
If such a test doesn't already exist, it would be good to also test that `list_versions` with `latest_only=True` actually works when the symbol exists.

Please could you also confirm whether there is good test coverage for `list_versions` without `latest_only`, and add some tests for it if they don't already exist.